### DISCUSSION TITLE
Any interest in Vercel Milvus for vector storage?

### DISCUSSION THREAD

**Discussion by fzliu** [vercel/community#2519] [Idea]
Hey folks - I'm here from Milvus https://github.com/milvus-io/milvus - we're an open source vector database commonly used for retrieval with LLMs, recommender systems, and unstructured data search. Trying to gauge interest in a Vercel vector store (a-la https://vercel.com/docs/storage) that utilizes Milvus.

If so, this is something we can contribute.

**Comment by mauceri** [comment]
Yes I could be interested in an implementation on a Vercel serverless function.

**Comment by hariskamran** [comment]
I am using Vercel for my LLM applications and recently started using Milvus for all my Vector Data.
It will be great if there is Milvus support in Vercel Storage.
